libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -69,6 +69,7 @@ export abstract class Bucket extends Resource {
       BucketInflightMethods.TRY_DELETE,
       BucketInflightMethods.SIGNED_URL,
       BucketInflightMethods.METADATA,
+      BucketInflightMethods.COPY,
```

Comment:
can you please add it to the target's bucket instead? (I believe you have a conflict here)